### PR TITLE
Tom1 login branch

### PR DESCRIPTION
**Pull request by TomHo521**
Added tests for all the login components with test coverage > 60%

Added a login failure condition for the Login component

### CODE REVIEW

**Review by saralandis — APPROVED** [review]

File: client/src/components/Login.jsx [hunk 1]
```diff
@@ -21,6 +22,7 @@ class Login extends React.Component {
     const validationPath = `http://localhost:8080/users/login?hostname=${this.state.hostName}&password=${this.state.password}`;
     axios.get(validationPath)
       .then(res => {
+        console.log(res);
```
Comment (dworthm) [inline client/src/components/Login.jsx:25]: Can you remove this console log?

File: db/controllers.js [hunk 2]
```diff
@@ -16,10 +16,12 @@ const getUsers = function() {
...
 // controller login route
 const login = function(obj_param) {
-  let { email } = obj_param;
-  let queryStr = `SELECT user_id, first_name, last_name, email FROM "BOC_Users" WHERE email = '${ email }'`;
+  let { email, password } = obj_param;
+  let queryStr = `SELECT user_id, first_name, last_name, email FROM "BOC_Users" WHERE email = '${ email }' AND password = '${ password }'`;
+  
   return pool.query(queryStr)
     .then(res => {
+      console.log('this is the res: ', res);
```
Comment (dworthm) [inline db/controllers.js:24]: Same - would you mind deleting this?

**Review by dworthm — APPROVED** [review]
Hey just a couple console logs to remove, but we can always delete those later. Nice tests!

File: db/controllers.js (hunk 2 above)
Comment (TomHo521) [reply db/controllers.js]: will do!

File: client/src/components/Login.jsx (hunk 1 above)
Comment (TomHo521) [reply client/src/components/Login.jsx]: yes

**Pull request merged**